content/articles/how-to-integrate-django-tailwind-styling-in-an-application/index.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-integrate-django-tailwind-styling-in-an-application/
+title: How to Integrate Django-Tailwind Styling in an Application
+description: This tutorial will discuss a better way to style Django applications using a framework called django-tailwind.
+author: umoh-mercy
+date: 2022-02-07T00:00:00-10:30
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-to-integrate-django-tailwind-styling-in-an-application/hero.png
+    alt: How to Integrate Django-Tailwind Styling in an Application Hero Image
+---
+Every application requires good styling for more readable content. Over the years, CSS (cascading style sheets) dominated before frameworks came into existence.
+<!--more-->
+In this tutorial, we will discuss a better way to style Django applications using a framework called `django-tailwind`. This makes use of the `tailwind` styling classes in Django projects.
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Project setup](#project-setup)
+- [Integrating django-tailwind](#integrating-django-tailwind)
+- [Making route for the project](#making-route-for-the-project)
+- [Views for the project](#views-for-the-project)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+To follow along with this tutorial, you should have:
+- An understanding of the Python programming language.
+- A pre-installed IDE, preferably [Visual Studio Code](https://code.visualstudio.com/download).
+- The [Python](https://python.org) programming language installed.
+- [Node.js](https://nodejs.org/en//) installed.
+- An understanding of the [Django](https://docs.djangoproject.com/en/4.0/) framework.
+
+### Project setup
+We will start this tutorial by setting up a project structure in Django's way of doing things. First off, we have to design a storage location for the project.
+
+Open your git bash, and type in the following commands:
+
+```bash
+cd
+cd Desktop
+mkdir project
+cd project
+```
+Check the working directory and ensure you are on the `Desktop` directory. You also made a directory called `project`, change to it.
```

Comment:
Hi @Dawe-7 

I am noticing cases like these where there is a mix of active and passive voices, which make the sentences illegible. Please change them to active voice as this is more engaging to the users and easier for them to follow through.

A good example:
 _-_ `You also made a directory called project, change to it`
 _+_ `Create a directory called project and navigate to it`